src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import duke.ui.MessageUi;
+
+
+/**
+ * Represents the main class that Ekud will run on.
+ */
+public class Duke {
+
+    private MessageUi ui;
+    private Storage storage;
+    private TaskList tasks;
+    private boolean f;
```

Comment:
Consider changing the name of this Boolean variable to a clearer one, according to the Java coding standard (for example doesFileExist)